code/iaas/model/src/main/java/io/cattle/platform/core/cleanup/TableCleanUp.java

```diff
@@ -37,6 +39,19 @@ public void cleanUp(){
                 .execute();
 
         long endProcessInstance = System.currentTimeMillis();
+
+        long startProcessExecution = System.currentTimeMillis();
+        long lowestProcessInstance = create()
```

Comment:
@hibooboo2 Can you explain the logic here?  Why do we consider the lowestProcessInstance, what are we trying to accomplish?